src/Illuminate/Cache/RedisLock.php

```diff
@@ -41,6 +41,20 @@ public function acquire()
         }
     }
 
+    /**
+     * Attempt to steal an existing lock.
+     *
+     * @return bool
+     */
+    public function steal()
+    {
+        if ($this->seconds > 0) {
+            return $this->redis->set($this->name, $this->owner, 'EX', $this->seconds, 'XX') == true;
+        } else {
+            return $this->redis->set($this->name, $this->owner, null, null, 'XX') == true;
+        }
```

Comment:
While I agree that early return is better, I'd prefer to leave this as-is because this function (steal) has the same structure as the one just before it (acquire).